cutils.c

```diff
@@ -1129,7 +1129,7 @@ void rqsort(void *base, size_t nmemb, size_t size, cmp_f cmp, void *opaque)
 
 /*---- Portable time functions ----*/
 
-#if defined(_MSC_VER)
+#if defined(_MSC_VER) || (__MINGW32__ && !defined(__MINGW64__))
```

Comment:
Come to think of it, we could use this everywhere on windows, perhaps that's a simpler thing, let's try it please.

Reply:
Yea I need to find a macro that mingw32 from Mingw-w64

Reply:
Ooh, allright, I could try

Reply:
See https://github.com/quickjs-ng/quickjs/blob/1eb9608d64242d0efec199b24f3bdf13063bf88e/libbf.h#L34 the size of limb_t is platform dependent, so perhaps an extra if-def is necessary to check.

Reply:
Thanks a lot, I will try this today as well as your `__MINGW32__` idea

Reply:
👍